### PR TITLE
RA-1387:Upgrade FontAwesome from 3.x to latest (5+)

### PR DESCRIPTION
**Pull request by haripriya999**
# RA-1387:Upgrade FontAwesome from 3.x to latest (5+)

## Description of what I changed
Upgraded font awesome to latest 5.9.0 to support a wider set of icons 

## Issue I worked on

see https://issues.openmrs.org/browse/RA-1387

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: omod/src/main/compass/sass/referenceapplication.scss [hunk 1]
```diff
@@ -3,4 +3,4 @@
 @import "font-awesome-5/fa-brands";
 @import "font-awesome-5/fa-regular";
 @import "font-awesome-5/fa-solid";
-@import "font-awesome-5/fontawesome"
\ No newline at end of file
+@import "font-awesome-5/fontawesome";
```
Comment (HerbertYiga) [inline omod/src/main/compass/sass/referenceapplication.scss]: did you really make any   change here?

Comment (HerbertYiga) [inline omod/src/main/compass/sass/referenceapplication.scss]: Did you really make any change here?

Comment (haripriya999) [reply omod/src/main/compass/sass/referenceapplication.scss]: I just placed the semicolon at the end.

**Comment by dkayiwa** [comment]
@haripriya999 i have tested your work but do not see any changes to the icons listed here? https://demo.openmrs.org/openmrs/uicommons/icons.page

**Comment by haripriya999** [comment]
@dkayiwa The icons listed here are of font awesome 3. I didn't know we had this page to display icons. In this PR i just upgraded font awesome to 5.9.0 and tested it by replacing a few old icons with the newly defined one's, and it worked. I didn't write code to display the font awesome 5 icons list in this page. Should I list them over here ?

**Comment by dkayiwa** [comment]
I would say yes.

**Comment by lnball** [comment]
@mogoodrich I've built my local dev environment watching the uicommons file changes, but it's still not picking up the fa-* icons.  Is there some change to the Mirebalais modules/build needed?  Maybe some part of these RefApp module modifications?

**Pull request merged**